src/components/Sidebar.js

```diff
@@ -134,6 +137,19 @@ export default ({
     { location: '/faq', text: 'FAQ' },
     { location: '/sponsors', text: 'SPONSORS' },
   ]
+  const [sponsors, setSponsors] = useState([])
+
+  useEffect(() => {
+    getSponsors().then(docs => {
+      // Only keep platinum tier sponsors for sidebar
+      const filteredDocs = docs
+        .filter(doc => doc.data().tier && doc.data().tier.toLowerCase() === 'platinum')
+        .map(doc => {
+          return doc.data()
+        })
```

Comment:
sorry one more LMAO implicit return

```suggestion
        .map(doc => doc.data())
```

Reply:
oops :^) c0fd4aa